core/service-test-runner/cli.js

```diff
@@ -59,6 +64,9 @@ const Runner = require('./runner')
 
 require('../unhandled-rejection.spec')
 
+const retry = {}
+retry.count = process.env.RETRY_COUNT || 0
```

Comment:
Will these need to be explicitly cast to numbers/integers? I'm not familiar with IcedFrisby's retry, but I've run into issues with env vars being stringified and then libs that are expecting numbers bark at those values

Reply:
[Documentation](https://github.com/IcedFrisby/IcedFrisby/blob/master/API.md#retrycount-backoff) says that count and backoff should be numbers, but `retry` works with strings. Nevertheless, I converted it to integers.